.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,38 @@
+# Bump Version workflow that is triggered manually
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      pcluster-version:
+        description: 'The target version of ParallelCluster CLI'
+        required: true
+        type: string
+      branch:
+        description: 'The Github branch name'
+        required: true
+        type: string
+
+jobs:
+  create-pull-requests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        ref: ${{ inputs.branch }}
+    - name: Modifiy Code to Change version
+      run: ./util/bump-version.sh ${{ inputs.pcluster-version }}
+        
+    - name: Create a Pull Request
+      uses: peter-evans/create-pull-request@v6
```

Comment:
Sorry I didnt catch it before. But are we sure we want to use a 3rd party git-repo and not this one https://github.com/marketplace/actions/github-action-for-creating-pull-requests?

Reply:
The current one is more poplular https://github.com/marketplace/actions/create-pull-request

I think both of them are third party

Reply:
I found the link for the one you are using https://github.com/marketplace/actions/create-pull-request